File: sketches/sketches.model/src/main/java/edu/kit/kastel/lissa/sketches/model/impl/elements/uml/UMLClass.java
package edu.kit.kastel.lissa.sketches.model.impl.elements.uml;

import edu.kit.kastel.lissa.sketches.model.elements.uml.IUMLClass;
import edu.kit.kastel.lissa.sketches.model.types.SketchBoxTypes;

public class UMLClass extends UMLThing implements IUMLClass {
	private static final long serialVersionUID = 3310996720085779684L;

	public UMLClass() {
		// NOP
	}

	@Override
	public SketchBoxTypes getCurrentInterpretation() {
		return SketchBoxTypes.CLASS;
	}

}

File: sketches/model/src/main/kotlin/edu/kit/kastel/lissa/sketches/model/impl/elements/uml/UMLClass.kt
package edu.kit.kastel.lissa.sketches.model.impl.elements.uml

import edu.kit.kastel.lissa.sketches.model.elements.uml.IUMLClass
import edu.kit.kastel.lissa.sketches.model.types.SketchBoxTypes

class UMLClass : UMLThing(), IUMLClass {
    override fun currentInterpretation() = SketchBoxTypes.CLASS
}